### PR TITLE
Remove Atlas references from getting-started pages

### PR DESCRIPTION
**Pull request by rcgenova**
I replaced references to Atlas auto-join with references to AWS and GCE auto-discovery in https://www.consul.io/intro/getting-started/join.html.

I removed references to the Atlas-hosted UI from https://www.consul.io/intro/getting-started/ui.html.

### CODE REVIEW

File: website/source/intro/getting-started/join.html.markdown [hunk 1]
```diff
@@ -141,29 +141,7 @@ learn about <em>one existing member</em>. After joining the cluster, the
...
-configuration.
-
-Alternatively, you can join a cluster at startup using the
-[`-join` flag](https://www.consul.io/docs/agent/options.html#_join) or
-[`start_join` setting](https://www.consul.io/docs/agent/options.html#start_join)
-with hardcoded addresses of other known Consul agents.
+Ideally, whenever a new node is brought up in your datacenter, it should automatically join the Consul cluster without human intervention. Consul facilitates auto-join by enabling the auto-discovery of instances in AWS or Google Cloud with a given tag key/value. To use the integration, add the [`retry_join_ec2`](https://www.consul.io/docs/agent/options.html?#retry_join_ec2) or the [`retry_join_gce`](https://www.consul.io/docs/agent/options.html?#retry_join_gce) nested object to your Consul configuration file. This will allow a new node to join the cluster without any hardcoded configuration. Alternatively, you can join a cluster at startup using the [`-join` flag](https://www.consul.io/docs/agent/options.html#_join) or [`start_join` setting](https://www.consul.io/docs/agent/options.html#start_join) with hardcoded addresses of other known Consul agents.
```
Comment (slackpad) [inline website/source/intro/getting-started/join.html.markdown]: These links can omit the leading `https://www.consul.io`, so just `(/docs/agent/options.html?#retry_join_ec2)`

File: website/source/intro/getting-started/ui.html.markdown [hunk 2]
```diff
@@ -13,39 +13,11 @@ box. UIs can be used for viewing all services and nodes, for viewing
...
-your credentials.
-
-## Self-hosted Dashboard
-
 <div class="center">
 ![Consul Web UI](consul_web_ui.png)
 </div>
 
-To set up the self-hosted UI, start the Consul agent with the
+To set up the [self-hosted UI](/downloads.html), start the Consul agent with the
```
Comment (slackpad) [inline website/source/intro/getting-started/ui.html.markdown]: The UI assets are built in (`-ui` is enough to get them), so we shouldn't point people over there by default. The downloaded UI assets are just for people who want to host elsewhere.

File: website/source/intro/getting-started/join.html.markdown (hunk 1 above)
Comment (rcgenova) [inline website/source/intro/getting-started/join.html.markdown]: Fixed.

File: website/source/intro/getting-started/ui.html.markdown (hunk 2 above)
Comment (rcgenova) [inline website/source/intro/getting-started/ui.html.markdown]: Fixed.

**Pull request merged**